added initial markup, created header and footer

Shevtsooov opened the pull request:
-[DEMO LINK](https://Shevtsooov.github.io/layout_miami/)

naziamoff reviewed (changes requested):
- sth is wrong with hovering items from the gallery, the hoverable area is huge horizontally

![image](https://user-images.githubusercontent.com/103033758/236502611-ea57b7e1-4222-41fd-813a-597a8a2e76fd.png)

- submitting a form redirects me to 405

Shevtsooov commented:
@naziamoff Hi, thanks for checking, corrected the images :hover, but I am not sure about the form submitting.
I have added the JS code for that. Please let me know if that is the proper fix.

silchyn reviewed (approved):
- Links in `Contact Us` do not change on hover
- Maybe we could scroll the user to the top after the form submission
